packages/flutter_android_lifecycle/README.md

```diff
@@ -0,0 +1,14 @@
+# flutter_android_lifecycle
+
+A new flutter plugin project.
```

Comment:
Are you planning to add the documentation later?

Reply:
Good catch. I actually had the READMEs reversed. Fixed that, and added a simple explanation for the example project.